src/app/modules/verification/verification.controller.ts

```diff
@@ -341,6 +381,15 @@ export const handleGenerateOtp = [
   _handleGenerateOtp,
 ] as ControllerHandler[]
 
+/**
+ * NOTE: Exported soley for testing
+ * @param updateData
+ * @returns
+ */
+export const updateHashForPaymentFieldWrapper = (
+  updateData: UpdatePaymentFieldData,
+) => VerificationModel.updateHashForPaymentField(updateData)
+
```

Comment:
nit: would these wrapper functions be better placed in `verification.util.ts` instead?

Reply:
Hm, maybe! I initially put them here as it is only used by the controller so there's no need for it to be exported. I think

Reply:
The wrappers can be removed with the refactoring I've done to shift the form / payment logic to the service layer, so this should be gone!